lib/ecs_deploy/task_definition.rb

```diff
@@ -1,8 +1,15 @@
 module EcsDeploy
+  RETRY_BACKOFF = lambda do |c|
```

Comment:
If you intend to define `EcsDeploy::RETRY_BACKOFF`, you should define it in lib/ecs_deploy.rb.
I prefer to define `EcsDeploy::TaskDefinition::RETRY_BACKOFF`.

Reply:
Oh, I made mistake...